packages/microsite/src/cli/microsite-build.ts

```diff
@@ -557,10 +569,79 @@ async function bundlePagesForSSR(paths: string[]) {
         entry.hydrateBindings = null;
       if (entry.hydrateStyleBindings.length === 0)
         entry.hydrateStyleBindings = null;
+      if (entry.hydrateBindings) {
+        for (const [file, exports] of Object.entries(entry.hydrateBindings)) {
+          const hydrated = hydrateMap[file.replace(/\.js$/, "")];
+          entry.hydrateBindings[file] = Object.keys(exports).reduce(
+            (acc, key) => {
+              return Object.assign(acc, { [hydrated[key] || key]: key });
+            },
+            {}
+          );
+        }
+      }
       return entry;
     });
 }
 
+const rewriteHydratedComponentDisplayNames = (): Plugin => {
```

Comment:
Likely the old plugin's name doesn't reflect its current responsibilities =)

Reply:
Ah good point, will update.